src/functions.php

```diff
@@ -270,10 +270,6 @@ function finder(): Finder
 
 function import(string $path): void
 {
-    if (!FunctionFinder::isInFindFunctions()) {
-        throw new \LogicException('The import function cannot be dynamically invoked, use it a the root of the PHP file.');
```

Comment:
@joelwurtz do you recall why you added this limitation?

Reply:
To have an error if a user import a function when running another one

Reply:
But what was the real limitation, why do you forbid that?

Reply:
Because it doesn't work since were are not in the discovering phase when running inside a function (and it doesn't make sense)